<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 78, 96)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-240</x>
      <y>40</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Showcard Gothic</family>
      <pointsize>50</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);

font: 50pt &quot;Showcard Gothic&quot;;
allign:centre;</string>
    </property>
    <property name="text">
     <string>RECIPES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="yes_radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);
font: 700 10pt &quot;Berlin Sans FB Demi&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="no_radio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);
font: 700 10pt &quot;Berlin Sans FB Demi&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>NO </string>
    </property>
   </widget>
   <widget class="QSlider" name="size_slider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="italian_check">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Italian</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chinese_check">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Chinese</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mexican_check">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mexican</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);
font: 700 18pt &quot;Berlin Sans FB Demi&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>VEGETARIAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recipe_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color: rgb(216, 216, 216);
  border: none;
  color: rgb(0, 0, 0);
  text-align: center; 
   border-radius: 30px;
font: 900 15pt &quot;Segoe UI Black&quot;;
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);
font: 700 18pt &quot;Berlin Sans FB Demi&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>NO. OF PEOPLE TO FEED</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(255, 255, 255);
font: 700 18pt &quot;Berlin Sans FB Demi&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>CUISINE</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>logo_label</zorder>
   <zorder>yes_radio</zorder>
   <zorder>no_radio</zorder>
   <zorder>size_slider</zorder>
   <zorder>italian_check</zorder>
   <zorder>chinese_check</zorder>
   <zorder>mexican_check</zorder>
   <zorder>lineEdit</zorder>
   <zorder>recipe_button</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
